NetJSON/NetJSON.Internals.cs

```diff
@@ -394,10 +394,6 @@ internal unsafe static object ToStringIfString(object value, NetJSONSettings set
 				dt = new DateTime(1970, 1, 1, 0, 0, 0, 0, DateTimeKind.Utc);
 
 				dt = dt.AddTicks(ticks * multiply);
-
-				if (timeZoneFormat == NetJSONTimeZoneFormat.Unspecified || timeZoneFormat == NetJSONTimeZoneFormat.Utc)
-					dt = dt.ToLocalTime();
```

Comment:
Why remove this line?

Reply:
This was causing test `TestSerializeDateUtcNowWithMillisecondDefaultFormatUtc` failure if run in UTC+3 environment. I believe this is due to DateTime constuctor with `kind` argument.

Reply:
I see. That is a good catch. I will run it locally tonight and get back to you. Thanks again for the effort.

Reply:
What condition can we put to make it call ToLocalTime properly? rather than removing it and requiring the caller to do this manually?

Reply:
I though about doing so yet haven't come up with any solution. Let me think about that again.